packages/react-core/src/components/Tabs/Tabs.tsx

```diff
@@ -17,7 +18,7 @@ export enum TabsComponent {
 
 export interface TabsProps extends Omit<React.HTMLProps<HTMLElement | HTMLDivElement>, 'onSelect'>, OUIAProps {
   /** Content rendered inside the tabs component. */
-  children: React.ReactNode;
+  children: React.ReactElement<TabProps>[];
```

Comment:
Looks ok. Don't know if this is considered a breaking change or not?

Reply:
Considering if you pass anything but `<Tab>` as a child of `<Tabs>` it doesn't render correctly I don't think it is. I get a weird spacer [when I pass a `<div>`](https://codesandbox.io/s/icy-feather-1n4fu?file=/index.js):
![image](https://user-images.githubusercontent.com/47335686/92638792-d75f5c80-f2a8-11ea-912d-713b4f9a30bc.png)

Reply:
I disagree Zack.  I think this would be a API breaking change regardless of how this was rendered.  Someone may be passing something other than`<Tabs>` and fixing the rendering with CSS.  This change would break their build.

Reply:
I just checked through all the [packages in our analytics](https://github.com/rh-uxd/patternfly-analytics) and all of them use `<Tabs><Tab /></Tabs>`. Not doing so is what I'd consider undefined behavior. This is what renders:
```html
<li class="pf-c-tabs__item">
  <button class="pf-c-tabs__link" id="pf-tab-undefined-pf-15997686294791nq1760spxn" aria-controls="pf-tab-section-undefined-pf-15997686294791nq1760spxn"></button>
</li>
```

I'm not sure how or why you could save that with CSS. If you pass anything other than a `Tab` our rendering ignores what you pass and outputs a weird `<li><button>`! If I'm a typescript consumer, having a correct type that tells me this would be very welcome.

Reply:
What if you want to make a custom tab to use inside of tabs.  I was thinking someone may want to do that and this could potential be a breaking change if they didn't use TabProps interface when they built their tab.

Reply:
Yes, if a Typescript consumer did that they would have to be explicit that their implementation matches `<Tab>`'s. For example, Kiali [already casts to `React.ReactElement<TabProps>`.](https://github.com/kiali/kiali-ui/blob/master/src/components/Tab/Tabs.tsx#L46)

As it is if we add props to `Tab` we will break these apps (and they _should_ break since they depend on our internals)!